### ISSUE TITLE
Introduction to Integral Theory

### ISSUE THREAD

**Issue by rufuspollock** [life-itself/community#123]
Write a short overview / introduction to Integral Theory and Ken Wilber.

**Comment by theo-cox** [comment]
See here for notes on one of the major books: https://lifeitself.us/2019/01/22/ken-wilber-integral-spirituality/

**Comment by rufuspollock** [comment]
@theo-cox and i have a bunch more notes on specific books that I can post (just writing some of them up into blog posts on specific books might be useful)

I also recommend a trawl of https://integrallife.com

**Comment by theo-cox** [comment]
@rufuspollock can you share the notes? This will be good to add to a content backlog

**Comment by rufuspollock** [comment]
>  can you share the notes? This will be good to add to a content backlog

@theo-cox done - just pushed in commit into the books subdirectory https://github.com/life-itself/community/commit/62b32b733fb956f832529fa9d7500bf9bb3083ae

**Comment by laurenwigmore** [comment]
@rufuspollock just flagging that this could be used as a beacon if we were to make it into an email course
#814

**Comment by rufuspollock** [comment]
@laurenwigmore yes, this could be a "learn" beacon. Have we got a beacon and/or learn epic we can add that point to ...

**Comment by laurenwigmore** [comment]
> @laurenwigmore yes, this could be a "learn" beacon. Have we got a beacon and/or learn epic we can add that point to ...

Yep, all already linked to the epic #826
